Error compiling newest master commit

I think the branch 'feature/window_focus' was not ready to be merged into master, as there is a compile error from a missing macro, but not in the other merge commit before that.

```
[ 25%] Building CXX object src/SFML/Window/CMakeFiles/sfml-window.dir/Win32/Wind
owImplWin32.cpp.obj
..\..\..\..\src\SFML\Window\Win32\WindowImplWin32.cpp: In member function 'virtu
al void sf::priv::WindowImplWin32::requestFocus()':
..\..\..\..\src\SFML\Window\Win32\WindowImplWin32.cpp:387:26: error: 'FLASHW_TRA
Y' was not declared in this scope
         info.dwFlags   = FLASHW_TRAY;
                          ^
src\SFML\Window\CMakeFiles\sfml-window.dir\build.make:486: recipe for target 'sr
c/SFML/Window/CMakeFiles/sfml-window.dir/Win32/WindowImplWin32.cpp.obj' failed
mingw32-make[2]: *** [src/SFML/Window/CMakeFiles/sfml-window.dir/Win32/WindowImp
lWin32.cpp.obj] Error 1
CMakeFiles\Makefile2:204: recipe for target 'src/SFML/Window/CMakeFiles/sfml-win
dow.dir/all' failed
mingw32-make[1]: *** [src/SFML/Window/CMakeFiles/sfml-window.dir/all] Error 2
Makefile:116: recipe for target 'all' failed
mingw32-make: *** [all] Error 2
```

```
$ g++ -v
Using built-in specs.
COLLECT_GCC=c:\TDM-GCC-32\bin\g++.exe
COLLECT_LTO_WRAPPER=c:/tdm-gcc-32/bin/../libexec/gcc/mingw32/4.8.1/lto-wrapper.e
xe
Target: mingw32
Configured with: ../../../src/gcc-4.8.1/configure --build=mingw32 --enable-langu
ages=ada,c,c++,fortran,lto,objc,obj-c++ --enable-libgomp --enable-lto --enable-g
raphite --enable-libstdcxx-debug --enable-threads=posix --enable-version-specifi
c-runtime-libs --enable-fully-dynamic-string --enable-libstdcxx-threads --enable
-libstdcxx-time --with-gnu-ld --disable-werror --disable-nls --disable-win32-reg
istry --disable-symvers --enable-cxx-flags='-fno-function-sections -fno-data-sec
tions -DWINPTHREAD_STATIC' --prefix=/mingw32tdm --with-local-prefix=/mingw32tdm
--with-pkgversion=tdm-2 --enable-sjlj-exceptions --with-bugurl=http://tdm-gcc.td
ragon.net/bugs
Thread model: posix
gcc version 4.8.1 (tdm-2)
```

--- a/src/SFML/Window/Win32/WindowImplWin32.cpp
+++ b/src/SFML/Window/Win32/WindowImplWin32.cpp
@@ -33,6 +33,7 @@
 #endif
 #define _WIN32_WINDOWS 0x0501
 #define _WIN32_WINNT   0x0501
+#define WINVER         0x0501
 #include <SFML/Window/Win32/WindowImplWin32.hpp>
 #include <SFML/Window/WindowStyle.hpp>
 #include <GL/gl.h>

